### PR TITLE
feat: ai chatbot integration poc

### PR DESCRIPTION
**Pull request by isekovanic**
### 🎯 Goal

Chat SDK PR: https://github.com/GetStream/stream-chat-react/pull/2570

### 🛠 Implementation details

_Provide a description of the implementation_

### 🎨 UI Changes

_Add relevant screenshots_

Make sure to test with both Angular and React (with both `MessageList` and `VirtualizedMessageList` components) SDKs

### CODE REVIEW

File: src/v2/styles/MessageInput/MessageInput-layout.scss [hunk 1]
```diff
@@ -129,6 +129,15 @@
     width: 100%;
   }
 
+  .str-chat__stop-generating-button {
+    background-image: var(--str-chat__circle-stop-icon);
+    width: calc(var(--str-chat__spacing-px) * 30);
+    height: calc(var(--str-chat__spacing-px) * 28);
+    cursor: pointer;
+    background-color: transparent;
+    border-width: 0;
```
Comment (MartinCupela) [inline src/v2/styles/MessageInput/MessageInput-layout.scss:138]: background-image, border, background-color and probably cursor belong to theme scss file. Could you please move them there?

File: src/v2/styles/AITypingIndicatorView/AITypingIndicatorView-theme.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@use '../utils';
```
Comment (MartinCupela) [inline src/v2/styles/AITypingIndicatorView/AITypingIndicatorView-theme.scss:1]: I believe we do not need this import as nothing from utils is used here

File: src/v2/styles/AITypingIndicatorView/AITypingIndicatorView-layout.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@use '../utils';
```
Comment (MartinCupela) [inline src/v2/styles/AITypingIndicatorView/AITypingIndicatorView-layout.scss:1]: I believe we do not need this import as nothing from utils is used here

Comment (isekovanic) [reply src/v2/styles/AITypingIndicatorView/AITypingIndicatorView-layout.scss:1]: Serves me right for copy pasting CSS files :D Thanks, will take care of it in a bit !

File: src/v2/styles/MessageInput/MessageInput-layout.scss (hunk 1 above)
Comment (isekovanic) [reply src/v2/styles/MessageInput/MessageInput-layout.scss:138]: You're absolutely right, I thought I did but it seems I forgot. Will do !

**Review by MartinCupela — APPROVED** [review]

**Pull request merged**